modules/gltf/editor/editor_import_blend_runner.cpp

```diff
@@ -281,6 +304,65 @@ Error EditorImportBlendRunner::do_import_rpc(const Dictionary &p_options) {
 	return OK;
 }
 
+bool EditorImportBlendRunner::_parse_blender_http_response(const Vector<uint8_t> &p_response_data, String &r_error_message) {
+	// Based on RPC Xml spec from: https://xmlrpc.com/spec.md
```

Comment:
I thought we we using jsonrpc?

Reply:
It's the first time I'm working in that section, but the response on my computer at least is clearly XML.
And the call is definitively done in XML:
![image](https://github.com/godotengine/godot/assets/81109165/56bc04be-aac6-496d-b1ca-39a3c06702b8)

Reply:
JSONRPC is used for the GDScript LSP, not for the Blender import. This indeed uses XMLRPC.